packages/github-actions/actions/prepare-node/README.md

```diff
@@ -24,9 +24,9 @@ jobs:
         uses: actions/checkout@v3
```

Comment:
Could you clarify why we're not updating all instances of `actions/checkout` to `v4`? Even if it's not required would it not be better to use the same across all actions?